corehq/apps/app_manager/suite_xml/features/case_tiles.py

```diff
@@ -19,6 +21,10 @@
 TILE_DIR = Path(__file__).parent.parent / "case_tile_templates"
 
 
+class CaseTileTemplates(models.TextChoices):
```

Comment:
Doesn't need to be in this PR, but I wonder if it'd be better to pull the other metadata, currently in files like [this](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml/case_tile_templates/person_simple.json) into python so that it's all in one place.

Reply:
What would the approach to translations be if the name moves into the json?

Reply:
Reading the filenames sounds okay, but I'm still stuck on the translation piece. For django to translate the strings, they have to appear somewhere in python (worst case in [static_strings](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/static_strings.py)). There's so little metadata in the json, if we have to put some of it in python we might be better off putting all of it in python, rather than having it in two places.